common/unified-chart/values.yaml

```diff
@@ -11,9 +11,29 @@ nameSuffix: v2
 
 argoRollouts:
   enabled: true
-  activeMetadata:
-    labels:
-      role: active
+  # activeMetadata:
```

Comment:
this will affect all services we need to be very carful

Reply:
activeMetadata needs to remain there - its needed when we dont activate progressiveDelivery